components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```
@@ -4658,6 +4735,7 @@ paths:
       security:
         - OAuth2Security:
             - apim:api_view
+            - apim:api_manage
```

It is better to use a new group for this as this is a seperate API.
```suggestion
            - apim:throttling_policy_manage
```